tox.ini

```diff
@@ -61,6 +62,7 @@ deps =
 basepython = {[qadocs]basepython}
 envdir = {[qadocs]envdir}
 commands =
+    python housekeep.py prep
```

Comment:
I figure it will be nice to verify which Python interpreter is actually being used by "qa" (and "docs" as well below).

Also, the env var dumping of `housekeep.py` might be valuable for troubleshooting these two testenvs.